apps/Makefile

```
@@ -0,0 +1,62 @@
+include Makefile.include
+
+init:								## Prepares local development environment.
+	cd ../ && make build-go
+	yarn install --immutable
+	cd ./pmm-ui && npm install
+
+dev:								## Starts local development.
+	make env-down
+	make env-up
+	yarn start
+
+dev-local-gs:								## Starts local development with local grafana server.
+	make env-down-local-gs
+	make env-up-local-gs
+	find ./.config/grafana/plugins ! -name '.gitkeep' -type f -exec rm -f {} +
+	docker cp pmm-grafana-dev:/srv/grafana/plugins/. ./.config/grafana/plugins/
+	find ./.config/grafana/workdir/conf ! -name '.gitkeep' -type f -exec rm -f {} +
+	docker cp pmm-grafana-dev:/usr/share/grafana/conf/. ./.config/grafana/workdir/conf/
+	find ./.config/grafana/percona-dashboards ! -name '.gitkeep' -type f -exec rm -f {} +
+	docker cp pmm-grafana-dev:/usr/share/percona-dashboards/. ./.config/grafana/percona-dashboards/
+	cp -fr ./.config/grafana/provisioning/ ./.config/grafana/workdir/conf/provisioning
+	rm -f ./.config/grafana/workdir/data/grafana.db
+	yarn start
+
+env-up:								## Starts devcontainer.
+	docker-compose -f docker-compose.dev.yaml build --no-cache; \
```

```suggestion
	docker compose -f docker-compose.dev.yaml build --no-cache; \
```
Why not use `docker compose`? It's already part of all latest docker distributions.